package.json

```diff
@@ -112,5 +113,10 @@
   "packageManager": "yarn@3.2.0",
   "workspaces": [
     "integration-tests"
-  ]
+  ],
+  "lint-staged": {
+    "**/*.js": [
+      "yarn eslint"
```

Comment:
Since many ESLint rules are configured as warnings instead of errors, this will let through many things that would fail on CI. This should include the `--max-warnings=0` flag the same as in CI.

https://github.com/DataBiosphere/terra-ui/blob/fac6806358f52c6df2cc6b3a3becab2b64a3d6ec/.circleci/config.yml#L194

Reply:
Done. Good catch! I'm wondering if we should remove that step in CircleCI since we're doing the same check on commit. It doesn't hurt to leave it as is.